tests/unit/test_token.py

```diff
@@ -4,6 +4,7 @@
 
 from dateutil.tz import tzutc
 
+from xrdsst.main import XRDSSTTest
```

Comment:
Lines 15 and 71: expected 2 blank lines, 1 found, reformat file

Reply:
What kind of tools are you using for this? If we wanna do some formatting check thing, we should embed this in the build process, not do comments here.

Changed.